gateway/api/services/file_storage.py

```diff
@@ -41,6 +44,21 @@ class FileStorage:  # pylint: disable=too-few-public-methods
         provider_name (str | None): name of the provider in caseis needed to build the path
     """
 
+    @staticmethod
+    def file_extension_is_valid(file_name: str) -> bool:
```

Comment:
```suggestion
    def is_valid_extension(file_name: str) -> bool:
```
Since it is part of the `FileStorage` you can omit `file` and since it is a boolean function, I always try to start with the verb.

Reply:
Done here: [cee8159](https://github.com/Qiskit/qiskit-serverless/pull/1547/commits/cee8159201661eb441795b6f3f37382899deee1c)